.storybook/static/storybook.css

```diff
@@ -258,3 +258,10 @@ body {
   background: #f7f7f7;
   color: black;
 }
+
+/* tmp class during dev phase for ReverseHighlight and reverseSnippet component */
+.ais-ReverseHighlight-highlighted,
```

Comment:
To give more context to the team here:

Designs for reverse highlighting are not setting background color like for regular highlighting. This would be awkward because by definition, multiple isolated parts can match.

![image](https://user-images.githubusercontent.com/6137112/100455319-c8759280-30be-11eb-9726-8e0208b1d8dd.png)

Designs usually only consist of a font color and/or font weight.

![image](https://user-images.githubusercontent.com/6137112/100455199-9cf2a800-30be-11eb-8cfc-8f8054573c6f.png)

It therefore might make sense to support a new CSS class in our [InstantSearch Specs](https://github.com/algolia/instantsearch-specs).